src/api/java/com/minecolonies/api/configuration/Configurations.java

```diff
@@ -34,8 +34,8 @@
             public  boolean allowInfiniteSupplyChests = false;
 
             @Config.RangeInt(min = (CITIZEN_RESPAWN_INTERVAL_MIN), max = CITIZEN_RESPAWN_INTERVAL_MAX)
```

Comment:
Should we change CITIZEN_RESPAWN_INTERVAL_MIN/MAX to take into account that we're using seconds instead of ticks now?

Reply:
The thing is, previously we were using seconds already but we put in way too much (we multiplied it by 20 in the code), therefore, when I went to test some code, the citizens needed like 5-10 minutes to spawn (each).